test/runtests.jl

```diff
@@ -41,5 +41,10 @@ module TestJLL end
         @test isdir(@eval TestJLL OpenLibm_jll.artifact_dir)
         @test isempty(@eval TestJLL OpenLibm_jll.PATH[])
         @test occursin(Sys.BINDIR, @eval TestJLL OpenLibm_jll.LIBPATH[])
+
+        # Issue #20
+        if Sys.iswindows()
+            @test occursin(Sys.BINDIR, JLLWrappers.get_julia_libpaths())
```

Comment:
This should be `in` rather than `occursin`
```suggestion
            @test Sys.BINDIR ∈ JLLWrappers.get_julia_libpaths()
```